src/utils/data.js

```diff
@@ -0,0 +1,26 @@
+import * as d3 from 'd3';
+let headings = [];
+let dataObject;
+
+//in the cuture if there are multiple databases we can provide which data to use as a parameter
```

Comment:
Future not cuture lol

Reply:
good point! can we go ahead and merge it for now and fix it later so to save some overhead?

Reply:
Sure no problem!